### ISSUE TITLE
Inconsistency between txnbuild.Timebounds and xdr.TimeBounds

### ISSUE THREAD

**Issue by drdgvhbh** [stellar/go#1476]
The "TimeBounds" struct name should be consistent between `xdr` and `txnbuild` packages. See the following files for reference.

Of course the xdr file is generated, in which case I propose changing the name in the `txnbuild` package.

https://github.com/stellar/go/blob/master/txnbuild/timebounds.go#L21

https://github.com/stellar/go/blob/master/xdr/xdr_generated.go#L7547

**Comment by ire-and-curses** [comment]
Yeah, that's a bit annoying, but so is breaking the API. We can consider this for the next major release.
